x/bank/module.go

```diff
@@ -59,6 +63,9 @@ func (AppModuleBasic) ValidateGenesis(cdc codec.JSONMarshaler, config client.TxE
 // RegisterRESTRoutes registers the REST routes for the bank module.
 func (AppModuleBasic) RegisterRESTRoutes(clientCtx client.Context, rtr *mux.Router) {
 	rest.RegisterHandlers(clientCtx, rtr)
+	// TODO: registerQueryHandler
+	// types.RegisterQueryHandlerFromEndpoint(context.Background(), &runtime.ServeMux{})
+	// types.RegisterQueryHandler(context.Background(), runtime.NewServeMux(), )
```

Comment:
I think this could be separated in its own PR.

In #5921, there's:
- point 2: add the grpc-gateway server
- point 3: add annotations on all .proto files

To have smaller PRs, I think this PR should be scoped to point 3

Reply:
Yes, @atheeshp may be we can just have annotations in this PR and keep the GRPC Gateway stuff in a separate PR. wdyt?

Reply:
I'm in favor of that ^

Reply:
Sure!